contentwarehouse/snippets/quickstart_sample.py

```diff
@@ -80,7 +81,8 @@ def quickstart(project_number: str, location: str) -> None:
 
     # Define Request
     create_document_request = contentwarehouse.CreateDocumentRequest(
-        parent=parent, document=document
+        parent=parent, document=document, request_metadata=contentwarehouse.RequestMetadata(
+            user_info=contentwarehouse.UserInfo(id=user_id))
```

Comment:
nit: spacing adjustment for readability
```suggestion
        parent=parent,
        document=document,
        request_metadata=contentwarehouse.RequestMetadata(
            user_info=contentwarehouse.UserInfo(id=user_id)
        )
```